docs/src/main/asciidoc/logging.adoc

```diff
@@ -580,14 +658,14 @@ And SLF4J:
     <artifactId>slf4j-jboss-logmanager</artifactId>
 </dependency>
 ----
-
++
 [source,gradle,role="secondary asciidoc-tabs-target-sync-gradle"]
 .build.gradle
 ----
 implementation("org.jboss.slf4j:slf4j-jboss-logmanager")
 ----
 
-NOTE: This is not needed for libraries that are dependencies of a Quarkus extension as the extension will take care of this for you.
+. Verify whether the logs generated by the added library adhere to the same format as the other Quarkus logs.
 
 [[loggingConfigurationReference]]
 == Logging configuration reference
```

Comment:
Should this section (and the other big included file with a similar properties list) be on its own reference page, (as per diataxis) and referenced from a how-to? Ignore if this is the plan for a later iteration. Thanks.